presto-password-authenticators/src/main/java/com/facebook/presto/password/LdapAuthenticatorFactory.java

```diff
@@ -41,6 +45,10 @@ public PasswordAuthenticator create(Map<String, String> config)
                     binder -> {
                         configBinder(binder).bindConfig(LdapConfig.class);
                         binder.bind(LdapAuthenticator.class).in(Scopes.SINGLETON);
+
+                        if (config.get(INTERNAL_LDAP_USER_CONFIG) != null) {
```

Comment:
It will bind the filter only to an app context you are trying to bootstrap here. To make it work you should bind this filter to every single http client in Presto core. This is totally different Guice context. And i'm not sure if it is available from inside of a plugin.